components/tag/stories/tag.stories.js

```diff
@@ -154,31 +169,37 @@ Emphasized.tags = ["!dev"];
 Emphasized.args = {
 	isEmphasized: true
 };
-
 Emphasized.parameters = {
 	chromatic: { disableSnapshot: true },
 };
 
-export const Invalid = TagGroups.bind({});
-Invalid.tags = ["!dev"];
-Invalid.args = {
-	isInvalid: true
-};
-
-Invalid.parameters = {
-	chromatic: { disableSnapshot: true },
-};
-
+/**
+ * Tags have the option to be removable or not. Removable tags have a small close ("x") button.
+ */
 export const Removable = TagsDefaultOptions.bind({});
+Removable.storyName = "Default, removable";
 Removable.tags = ["!dev"];
 Removable.args = {
 	hasClearButton: true,
 };
-
 Removable.parameters = {
 	chromatic: { disableSnapshot: true },
 };
 
+/**
+ * Tags have a read-only option for when content in the disabled state still needs to be shown.
+ * Read-only tags cannot be interacted with or changed.
+ */
+export const ReadOnly = TagsDefaultOptions.bind({});
+ReadOnly.storyName = "Read only";
+ReadOnly.tags = ["!dev"];
+ReadOnly.args = {
+	isReadOnly: true,
+};
+ReadOnly.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
```

Comment:
This is a follow-up item also, it wasn't in the design spec but has existed in the S1 design guidelines.

Reply:
I removed `user-select: none;` from the tag, so now the text is selectable (but not copy-able, is there a native way to do that in CSS?). Happy to restore it if that causes unintended issues though.

Reply:
Is the text still not copy-able for either of you? I can copy and paste it from a read-only tag.
<img width="147" alt="Screenshot 2025-05-07 at 5 04 28 PM" src="https://github.com/user-attachments/assets/95590f59-1adb-4d3c-b32f-5b1cb6f06a00" />

> Tag label that i copied

Reply:
I don't believe that functionality is native to HTML or CSS; it'd be a JavaScript implementation. I don't think I'm mentioning that in the docs anywhere but should I? I would hope that consumers would look at the design guidelines instead of solely trusting our Storybook documentation, but I know sometimes we mention that a tooltip or other thing that's outside of the scope of our implementation should be available.